src/NServiceBus.Core/Routing/ConfigureUniquelyAddressableQueue.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace NServiceBus
```

Comment:
There doesn't appear to be a config folder. Was there a different folder you meant?

I chose Routing initially because all of the test that use it are routing tests, and routing deals with instances.

The other option I was considering was to put it in the Transports folder so it could sit next to the Receiving feature that actually uses the `EndpointInstanceDiscriminator` setting.